src/metatensor/models/cli/conf/base.yaml

```diff
@@ -1,3 +1,3 @@
 device: "cpu"
-base_precision: 64
+base_precision: 32
```

Comment:
Uhhh are we sure we want this to be the default behavior? People have had issues with `float32` MD

Reply:
I just changed this to be consistent with torch. Which people had issues? The examples seems to be good and usually problems in MD appear if the forces are not summed up in double precision but as a base precision I think a normal float should be fine.

Reply:
Ok, we can discuss this together with other people. I still think float64 is a safer default

Reply:
I am using the power of omegaconf's custom resolvers to obtain the preferred dtype for each architecture. When we are happy with this I will do the same for the device (but in a different PR)